isisd/isis_tlvs.h

```diff
@@ -213,7 +213,7 @@ struct isis_router_cap {
 	/* RFC 8667 section #3 */
 	struct isis_sr_block srgb;
 	struct isis_sr_block srlb;
-	uint8_t algo[SR_ALGORITHM_COUNT];
+	bool algo[SR_ALGORITHM_COUNT];
 	/* RFC 8491 */
```

Comment:
better to not change what RFC says: "Algorithm:  1 octet of algorithm"
I don't know how different systems encode bool value.